custom/opm/reports.py

```diff
@@ -500,7 +500,7 @@ def join_rows(self, rows):
         if len(rows) == 1:
             return rows[0]
         def zip_fn((i, values)):
-            if i == self.column_index('num_children'):
+            if i == self.column_index('num_children') or i == self.column_index('year_end_bonus_cash'):
```

Comment:
Yeah, maybe `i` should be renamed `column_index` or converted into the column name for this type of dispatching.